pages/stories/about-us/index.html.md

```diff
@@ -10,13 +10,17 @@ description: Who we are, what we do, and how we do it
 blocks-njk:
   main: layouts/partials/articles/main.njk
 image:
-  src: https://placeimg.com/1280/720/arch
-  alt: Everything we do is driven by you
+  src: /images/articles/about-people.jpg
+  alt: Various Buildsters outside the Duke of York cinema in Brighton
   remote: true
 ogType: article
 section: stories
 ---
 ## Who we are
+<div class="web-c-video-player">
+  <iframe src="https://www.youtube-nocookie.com/embed/E_MIWWwnd60" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

Comment:
Should we add `?rel=0` to the src of this video to prevent third party "related" videos being shown at the end of our video? In theory, without it, they could be our competitors or content that doesn't sit well with the Wipro ethics.

Reply:
Will do. I wasn't aware of that option. Nice catch!